### PR TITLE
Migrates React app to new UI

### PR DESCRIPTION
**Pull request by Gargol**
refs #1110

### CODE REVIEW

**Comment by sindresorhus** [comment]
Nice! Thanks for helping out :)

-

These doesn't load though:

localhost/:18 GET http://localhost:8000/examples/react/node_modules/react/dist/react-with-addons.js 
localhost/:19 GET http://localhost:8000/examples/react/node_modules/react/dist/JSXTransformer.js 
localhost/:20 GET http://localhost:8000/examples/react/node_modules/director/build/director.js

File: examples/react/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+	"todomvc-common": "~0.3.0",
+	"director": "~1.2.0",
+	"react": "~0.12.0",
```
Comment (sindresorhus) [inline examples/react/package.json]: `~` => `^`

Comment (arthurvr) [inline examples/react/package.json]: Also, package.json will always use 2 spaces indentation.

**Comment by Gargol** [comment]
should be a lot better now :smile:

**Comment by sindresorhus** [comment]
Landed. Woot :)

**Pull request merged**